src/people/widgetViews/summaries/wantedSummaries/__tests__/CodingBounty.spec.tsx

```diff
@@ -446,4 +446,57 @@ describe('MobileView component', () => {
       await waitFor(() => expect(screen.getByText('complete')));
     })();
   });
+
+  it('displays the confirmation screen when the "Pay Bounty" button is clicked', () => {
+    const props: CodingBountiesProps = {
+      ...defaultProps,
+      creatorStep: 0,
+      paid: false,
+      isAssigned: true
+    };
+
+    const { getByText, queryByText } = render(<MobileView {...props} />);
+
+    const payBountyButton = getByText('Pay Bounty');
+    fireEvent.click(payBountyButton);
+
+    expect(getByText('Are you sure you want to')).toBeInTheDocument();
+    expect(queryByText('Pay this Bounty?')).toBeInTheDocument();
+  });
+
+  it('marks the bounty as paid when the confirmation screen "Yes" button is clicked', async () => {
+    const mockOpenPaymentConfirmation = jest.fn();
+    const mockMakePayment = jest.fn();
+
+    jest.mock('../../../../components/common', () => ({
```

Comment:
I think this path may be incorrect
should be
`../../../../../components/common`